### PR TITLE
toggle language for certificate view when prefer language is not en

### PR DESCRIPTION
**Pull request by wjllance**
add a toggle button when preferred language is not en
when click toggle button, it will convert translation between en and preferredLanguage

### CODE REVIEW

File: app/views/user/CertificatesView.coffee [hunk 1]
```diff
@@ -59,6 +63,7 @@ module.exports = class CertificatesView extends RootView
     @listenToOnce @courseLevels, 'sync', @calculateStats
 
     @certificateNumber = @hashString(@user.id + @courseInstanceID)
+    @needToggle = @nowLang.split('-')[0] != 'en'
```
Comment (AndrewJakubowicz) [inline app/views/user/CertificatesView.coffee]: Maybe the name could be more descriptive? I.e.`needLanguageToggle`?

File: app/views/user/CertificatesView.coffee [hunk 2]
```diff
@@ -10,13 +10,17 @@ ThangTypeConstants = require 'lib/ThangTypeConstants'
 ThangType = require 'models/ThangType'
 utils = require 'core/utils'
 fetchJson = require 'core/api/fetch-json'
+locale = require 'locale/locale'
 
 module.exports = class CertificatesView extends RootView
   id: 'certificates-view'
   template: require 'templates/user/certificates-view'
+  nowLang: me.get('preferredLanguage', true)
```
Comment (AndrewJakubowicz) [inline app/views/user/CertificatesView.coffee]: Should we instantiate this in `intialize`?
I don't think we should be setting this on a prototype of the `CertificatesView`.

File: app/views/user/CertificatesView.coffee [hunk 3]
```diff
@@ -10,13 +10,17 @@ ThangTypeConstants = require 'lib/ThangTypeConstants'
...
+  needToggle: false
```
Comment (AndrewJakubowicz) [inline app/views/user/CertificatesView.coffee]: Similarly this should be set in `initialize`.

**Comment by wjllance** [comment]
> Looks great! I also manually tested and it works fantastically.
> 
> Comments are mostly nitpicks around possibly improving naming of some variables.
> We should not be assigning dynamic variables to the prototype of the class. Please initialize the new fields in the `initialize` method.

Got it! Updated

**Pull request merged**